// @prettier/plugin-pug/dist/options/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.options = exports.CATEGORY_PUG = void 0;
const attribute_separator_1 = require("./attribute-separator");
const attribute_sorting_1 = require("./attribute-sorting");
const closing_bracket_position_1 = require("./closing-bracket-position");
const comment_preserve_spaces_1 = require("./comment-preserve-spaces");
const common_1 = require("./common");
const empty_attributes_1 = require("./empty-attributes");
const pug_class_notation_1 = require("./pug-class-notation");
const pug_explicit_div_1 = require("./pug-explicit-div");
const pug_framework_1 = require("./pug-framework");
const pug_id_notation_1 = require("./pug-id-notation");
const pug_single_file_component_indentation_1 = require("./pug-single-file-component-indentation");
const wrap_attributes_1 = require("./wrap-attributes");
exports.CATEGORY_PUG = 'Pug';
exports.options = {
    pugPrintWidth: common_1.PUG_PRINT_WIDTH_OPTION,
    pugSingleQuote: common_1.PUG_SINGLE_QUOTE_OPTION,
    pugTabWidth: common_1.PUG_TAB_WIDTH_OPTION,
    pugUseTabs: common_1.PUG_USE_TABS_OPTION,
    pugBracketSpacing: common_1.PUG_BRACKET_SPACING_OPTION,
    pugArrowParens: common_1.PUG_ARROW_PARENS_OPTION,
    pugSemi: common_1.PUG_SEMI_OPTION,
    attributeSeparator: attribute_separator_1.ATTRIBUTE_SEPARATOR_OPTION,
    pugAttributeSeparator: attribute_separator_1.PUG_ATTRIBUTE_SEPARATOR_OPTION,
    closingBracketPosition: closing_bracket_position_1.CLOSING_BRACKET_POSITION_OPTION,
    pugClosingBracketPosition: closing_bracket_position_1.PUG_CLOSING_BRACKET_POSITION_OPTION,
    commentPreserveSpaces: comment_preserve_spaces_1.COMMENT_PRESERVE_SPACES_OPTION,
    pugCommentPreserveSpaces: comment_preserve_spaces_1.PUG_COMMENT_PRESERVE_SPACES_OPTION,
    pugSortAttributes: attribute_sorting_1.PUG_SORT_ATTRIBUTES_OPTION,
    pugSortAttributesBeginning: attribute_sorting_1.PUG_SORT_ATTRIBUTES_BEGINNING_OPTION,
    pugSortAttributesEnd: attribute_sorting_1.PUG_SORT_ATTRIBUTES_END_OPTION,
    pugWrapAttributesThreshold: wrap_attributes_1.WRAP_ATTRIBUTES_THRESHOLD,
    pugWrapAttributesPattern: wrap_attributes_1.WRAP_ATTRIBUTES_PATTERN,
    pugEmptyAttributes: empty_attributes_1.PUG_EMPTY_ATTRIBUTES_OPTION,
    pugClassNotation: pug_class_notation_1.PUG_CLASS_NOTATION,
    pugIdNotation: pug_id_notation_1.PUG_ID_NOTATION,
    pugExplicitDiv: pug_explicit_div_1.PUG_EXPLICIT_DIV,
    pugEmptyAttributesForceQuotes: empty_attributes_1.PUG_EMPTY_ATTRIBUTES_FORCE_QUOTES_OPTION,
    pugSingleFileComponentIndentation: pug_single_file_component_indentation_1.PUG_SINGLE_FILE_COMPONENT_INDENTATION,
    pugFramework: pug_framework_1.PUG_FRAMEWORK
};
